python/demo/demo_lagrange_variants.py

```diff
@@ -176,10 +176,11 @@ def saw_tooth(x):
             for i in range(51):
                 pts.append([cell / 10 + i / 50 / 10, 0, 0])
                 cells.append(cell)
-        pts = np.array(pts)
-        values = uh.eval(pts, cells)
-        plt.plot(pts[:, 0], [saw_tooth(i[0]) for i in pts], "k--")
-        plt.plot(pts[:, 0], values, "r-")
+        pts_np = np.array(pts)
+        cells_np = np.array(cells)
+        values = uh.eval(pts_np, cells_np)
```

Comment:
I don't see why we shouldn't allow lists.

Reply:
Will change.